Fixed #29490 -- Added support for object-based Media CSS and JS paths. (#15349)
django/django: docs/topics/forms/media.txt

    <script src="https://static.example.com/animations.27e20196a850.js"></script>
    <script src="http://othersite.com/actions.js"></script>

Paths as objects
----------------

.. versionadded:: 4.1

Asset paths may also be given as hashable objects implementing an
``__html__()`` method. The ``__html__()`` method is typically added using the

> :func:`~django.utils.html.html_safe` decorator. The object is responsible to
> output the complete HTML ``<script>`` or ``<link>`` tag content::

Reviewer: ```suggestion
:func:`~django.utils.html.html_safe` decorator. The object is responsible for
outputting the complete HTML ``<script>`` or ``<link>`` tag content::
```
